EasyPost/Models/API/Shipment.cs

```diff
@@ -154,7 +154,7 @@ public async Task Buy(Rate rate, string? insuranceValue = null, bool withCarbonO
         /// </summary>
         /// <param name="fileFormat">Format to generate the label in. Valid formats: "pdf", "zpl" and "epl2".</param>
         [CrudOperations.Update]
-        public async Task<Shipment> GenerateLabel(string fileFormat)
+        public async Task<Shipment> GenerateLabel(string fileFormat = "pdf")
```

Comment:
Question: Why have a default instead of requiring users to specify?

Reply:
We can do that too. We have a default on batch generate label, so just want to remain consistent either way.

Reply:
I don't love that we have one on batch generate label... I'd prefer we didn't include them but we can't just rip out the batch one.

Do we do this in any of our other libs?

Reply:
Java and Python have no default, so we can go with that.